aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -58,13 +58,29 @@ Tensor& add_(Tensor& self, const Tensor& other, Scalar alpha) {
 }
 
 Tensor& div_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  if (isIntegralType(result.scalar_type(), /*includeBool=*/ true)) {
+    TORCH_WARN_ONCE(
+      "Integer division of tensors using div or / is deprecated. ",
+      "In a future release div and / will ",
+      "perform true division as in Python 3. Use true_divide or ",
+      "floor_divide instead.");
```

Comment:
nit: should we say "or //" (it's shorter).

Reply:
I mean `floor_divide (or // in Python)`

Reply:
Sounds good.